packages/flutter_tools/templates/module/ios/host_app_ephemeral_cocoapods/Podfile.copy.tmpl

```diff
@@ -3,6 +3,9 @@ platform :ios, '8.0'
 target 'Runner' do
   flutter_application_path = '../'
```

Comment:
This doesn't seem like it does anything with the new global `$flutter_application_path`.